tool/deploy.dart

```diff
@@ -0,0 +1,60 @@
+import 'dart:io';
+
+import 'package:grinder/grinder.dart';
+import 'package:pub_semver/pub_semver.dart';
+
+main(List args) => grind(args);
+
+@DefaultTask('Deploy software')
+deploy() {
```

Comment:
You may want to just add this task to the main `tool/grind.dart` script instead of creating a new library. It can support multiple tasks, with or without dependencies on the other tasks. Run `grind -h` to see the list of defined tasks, and `grind <task>` to run a specific one.